src/main/java/Weng.java

```diff
@@ -0,0 +1,160 @@
+
+public class Weng {
+    private static Task[] tasks = new Task[100];
+    private static int numItem = 0;
+
+    public static void main(String[] args) {
+        greeting();
+        inputHandler();
+    }
+
+    public static Task[] getTasks() {
+        return tasks;
+    }
+
+    public static void setTasks(Task[] tasks) {
+        Weng.tasks = tasks;
+    }
+
+    public static void addTodo(String[] currLine) {
+        String description = String.join(" ", Arrays.copyOfRange(currLine, 1, currLine.length));
+        addAndPrintHandler(new Todo(description));
+    }
+
+    public static void addEvent(String[] input) {
+        addAndPrintHandler(new Event(input));
+    }
+
+    public static void addDeadline(String[] input) {
+
+        addAndPrintHandler(new Deadline(input));
+    }
+
+    public static void setIndexInTask(Task task, int index) {
+        Weng.tasks[index] = task;
+    }
+
+    public static int getNumItem() {
+        return numItem;
+    }
+
+    public static void setNumItem(int numItem) {
+        Weng.numItem = numItem;
+    }
+
+    private static void addAndPrintHandler(Task newTask) {
+        setIndexInTask(newTask, getNumItem());
+        setNumItem(getNumItem() + 1);
+        print("Got it. I've added this task:");
+        print("\t" + newTask.toString());
+        print("Now you have " + getNumItem() + " tasks in the list.");
+    }
+
+    public static void print(String words) {
+        System.out.println("\t" + words);
+    }
+
+    public static void print_line() {
+        print("____________________________________________________________");
+    }
+
+    private static void inputHandler() {
+        Scanner scanner = new Scanner(System.in);
+        String line;
+        loop:
+        while (true) {
+            line = scanner.nextLine();
+            String[] currLine = line.split(" ");
+            print_line();
+            switch (currLine[0]) {
+            case "bye":
+                goodbye();
+                break loop;
+            case "list":
+                listTasks();
+                break;
+            case "todo":
+                addTodo(currLine);
+                break;
+            case "deadline":
+                addDeadline(currLine);
+                break;
+            case "event":
+                addEvent(currLine);
+                break;
+            case "unmark":
+                unmarkTask(currLine);
+                break;
+            case "mark":
+                markTask(currLine);
+                break;
+            default:
+                print("Invalid command");
+                break;
+            }
+            print_line();
+        }
+        print_line();
+    }
+
+    public static void unmarkTask(String[] currLine) {
+        int index = parseInt(currLine[1]);
+        if (getNumItem() < index) {
+            print("Index out of range");
+        } else {
+            if (!getTasks()[index].isDone()) {
+                print("Already unmarked");
```

Comment:
is it possible to not embed your if-else inside another else statement? think of other ways to represent this because it makes it difficult to understand what is going on